task5/terraform.tfvars

```diff
@@ -0,0 +1,13 @@
+aws_region   = "us-east-1"
+prefix_name  = "this"
+environment  = "lab"
+pub_key_name = "MyKeyPair"

```

Comment:
Please handle aws_key_pair creation within your module.  you can place it to root component.

Reply:
195179aa071530fdc9de39c4d57f6e316459259b added